src/content/1.7/modules/concepts/templating/import-js.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to import core js files 
+menuTitle: Import core js files
```

Comment:
Maybe a bit more formal? 😉 

```suggestion
menuTitle: Import Core JavaScript files
```

Reply:
Maybe it's too long for the menu?

Reply:
Looks fine tbh 

![image](https://user-images.githubusercontent.com/14963751/125924356-5ebdfd44-f502-492c-a907-1de571d5f90c.png)